src/webhooks/types.ts

```diff
@@ -33,7 +40,13 @@ export interface WebhookRegistryEntry {
 interface WebhookCheckResponseNode {
   node: {
     id: string;
-    callbackUrl: string;
+    endpoint: {
```

Comment:
Just noting that this change probably means we'd need a major release since after this since it'll break existing apps.

Reply:
Yeah, this is where I had some reservations. I don't think it would be too much effort to write some conditional logic to check against a given API version if you think it's worth doing? I noticed this library is backwards compatible for quite a few versions, so it seems like it would be a shame to drop that just for this feature.